themes/digital.gov/src/js/common/subscribe.js

```diff
@@ -0,0 +1,8 @@
+// scroll form into view when on /subscribe page
+const subscribeButton = document.querySelector(".subscribe-button");
+const signupForm = document.getElementById("#newsletter-signup ");
```

Comment:
This method takes the ID without the hash.

```suggestion
const signupForm = document.getElementById("newsletter-signup ");
```